mula/tests/integration/test_services.py

```diff
@@ -0,0 +1,194 @@
+import copy
+import time
+import unittest
+from unittest import mock
+
+from scheduler import config, models, storage
+from scheduler.connectors import services
+from scheduler.utils import remove_trailing_slash
+
+from tests.factories import PluginFactory
+
+
+class BytesTestCase(unittest.TestCase):
+    def setUp(self) -> None:
```

Comment:
Since there's no teardown of the tests (mainly "logging" the user out or clearing the session), I assume creating the new settings and client instance during test setup results in new login sessions?

Reply:
Yes, that was my thinking as well